### DISCUSSION TITLE
Pages (de)activated on Github: not more responsive to VS Code modifications

### DISCUSSION THREAD

**Discussion by pajaro1966** [squidfunk/mkdocs-material#7703] [Q&A]
Sorry for this basic post, but I'm stuck. I've created a Material Mkdocs site on VS Code and uploaded it to GitHub Pages. I control everything from VS Code and don't modify anything directly on Github. However, I have just disabled and enabled Pages directly from Github manually (terrible idea) and now when making changes on VS Code those changes don't modify the Github repository and therefore Github Pages. I have no idea how to reconnect my version on VS Code with the one on Github.
I got a message saying that I have "divergent branches", but don't know how to solve this.
Thanks in advance for your help.

Antonio

**Comment by kamilkrzyskow** [comment]
Hello @pajaro1966,
sorry about the late response, I hope the issue has been resolved.

I don't know how you've setup the site on VS Code to automatically upload to GitHub Pages, as this isn't really documented anywhere in the docs. Do you have a repository link where we could see the deployed files?

My assumption is that you initially deployed the `gh-pages` branch from your local PC, and then did something to recreate it from GitHub Pages, so your local version of the `gh-pages` isn't synced with the remote branch online, hence the "divergent branches" issue.

The quickest fix would be to remove the `gh-pages` branch entirely on GitHub and then redeploy it from your local PC from VS Code. I'm using the default `gh-pages` name for the branch with the HTML files, it could be named differently. I'm not talking about the `main`/`master` branch with your `mkdocs.yml` and `docs` Markdown files, this should stay as-is.
